modules/test/playwright/tests/client-extension-web/env/portal-ext.properties

```diff
@@ -1 +1,3 @@
+feature.flag.LPD-27222=true
+
```

Comment:
Do we need to put this FF here? In most cases, just `featureFlagsTest` fixture is enough. This should just be needed if FF needs to be set on DXP startup. 

If it is needed here, I don't think we need the fixture then.

Reply:
Ahh you are right, I don't need the fixture. Thanks @markocikos